src/SSHDebugPS/Docker/DockerHelper.cs

```diff
@@ -58,9 +59,10 @@ public static IEnumerable<DockerContainerInstance> GetLocalDockerContainers(stri
                         }
                         else
                         {
-                            var containerInstance = DockerContainerInstance.Create(args);
-                            if (containerInstance != null)
```

Comment:
think about updating the text in the status on the window with something like `Found x containers, Failed to parse y`. or even creating a status message that will tell users to go look in the output window.